icon-registry/icons/technology-dashboard-running_x48.svg

```diff
@@ -0,0 +1,14 @@
+<svg viewBox="0 0 48 48" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M40 8H8C6.34315 8 5 9.34315 5 11V16H43V11C43 9.34315 41.6569 8 40 8Z" fill="#E1E3ED" class="icon-light"/>
+<path fill-rule="evenodd" clip-rule="evenodd" d="M8 8C6.34315 8 5 9.34315 5 11V16H43V11C43 9.34315 41.6569 8 40 8H8Z" fill="#E1E3ED" class="icon-light"/>
+<path d="M19 30L14 28L9 31V36H38C38.5523 36 39 35.5523 39 35V23L34 24L29 27L24 25L19 30Z" fill="#E1E3ED" class="icon-light"/>
+<path d="M19 12H39M43 11V37C43 38.6569 41.6569 40 40 40H24H16H8C6.34315 40 5 38.6569 5 37V11M43 11C43 9.34315 41.6569 8 40 8H8C6.34315 8 5 9.34315 5 11M43 11V16H5V11M39 23V35C39 35.5523 38.5523 36 38 36H9M39 23V21C39 20.4477 38.5523 20 38 20H10C9.44772 20 9 20.4477 9 21V36M39 23L34 24L29 27L24 25L19 30L14 28L9 31V36" stroke="#9095AD" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="icon-dark"/>
+<path d="M10 12C10 12.5523 9.55228 13 9 13C8.44772 13 8 12.5523 8 12C8 11.4477 8.44772 11 9 11C9.55228 11 10 11.4477 10 12Z" fill="#9095AD" class="icon-light"/>
+<path d="M13 12C13 12.5523 12.5523 13 12 13C11.4477 13 11 12.5523 11 12C11 11.4477 11.4477 11 12 11C12.5523 11 13 11.4477 13 12Z" fill="#9095AD" class="icon-light"/>
+<path d="M16 12C16 12.5523 15.5523 13 15 13C14.4477 13 14 12.5523 14 12C14 11.4477 14.4477 11 15 11C15.5523 11 16 11.4477 16 12Z" fill="#9095AD" class="icon-light"/>
+<path d="M39 36.9993C39 41.4175 35.4183 44.9993 31 44.9993C26.5817 44.9993 23 41.4175 23 36.9993C23 32.581 26.5817 28.9993 31 28.9993C35.4183 28.9993 39 32.581 39 36.9993Z" fill="currentColor" class="icon-light"/>
+<path d="M37 36.9993C37 40.313 34.3137 42.9993 31 42.9993C27.6863 42.9993 25 40.313 25 36.9993C25 33.6856 27.6863 30.9993 31 30.9993C34.3137 30.9993 37 33.6856 37 36.9993Z" fill="currentColor" class="icon-light" />
+<path fill-rule="evenodd" clip-rule="evenodd" d="M31 31.9993C28.2386 31.9993 26 34.2378 26 36.9993C26 39.7607 28.2386 41.9993 31 41.9993C33.7614 41.9993 36 39.7607 36 36.9993C36 34.2378 33.7614 31.9993 31 31.9993ZM24 36.9993C24 33.1333 27.134 29.9993 31 29.9993C34.866 29.9993 38 33.1333 38 36.9993C38 40.8653 34.866 43.9993 31 43.9993C27.134 43.9993 24 40.8653 24 36.9993Z" fill="#C5C9FD" class="icon-light-secondary"/>
+<path fill-rule="evenodd" clip-rule="evenodd" d="M30 30.9993C30 30.447 30.4477 29.9993 31 29.9993C34.866 29.9993 38 33.1333 38 36.9993C38 37.5516 37.5523 37.9993 37 37.9993C36.4477 37.9993 36 37.5516 36 36.9993C36 34.2378 33.7614 31.9993 31 31.9993C30.4477 31.9993 30 31.5516 30 30.9993Z" fill="#4956E3" class="icon-dark-secondary"/>
+<path d="M36 36.9993C36 39.7607 33.7614 41.9993 31 41.9993C28.2386 41.9993 26 39.7607 26 36.9993C26 34.2378 28.2386 31.9993 31 31.9993C33.7614 31.9993 36 34.2378 36 36.9993Z" fill="currentColor" class="icon-light"/>
```

Comment:
I think the idea is that these are supposed to be white always? But I'm not 100% sure.

```suggestion
<path d="M39 36.9993C39 41.4175 35.4183 44.9993 31 44.9993C26.5817 44.9993 23 41.4175 23 36.9993C23 32.581 26.5817 28.9993 31 28.9993C35.4183 28.9993 39 32.581 39 36.9993Z" fill="white"/>
<path d="M37 36.9993C37 40.313 34.3137 42.9993 31 42.9993C27.6863 42.9993 25 40.313 25 36.9993C25 33.6856 27.6863 30.9993 31 30.9993C34.3137 30.9993 37 33.6856 37 36.9993Z" fill="currentColor" class="icon-light" />
<path fill-rule="evenodd" clip-rule="evenodd" d="M31 31.9993C28.2386 31.9993 26 34.2378 26 36.9993C26 39.7607 28.2386 41.9993 31 41.9993C33.7614 41.9993 36 39.7607 36 36.9993C36 34.2378 33.7614 31.9993 31 31.9993ZM24 36.9993C24 33.1333 27.134 29.9993 31 29.9993C34.866 29.9993 38 33.1333 38 36.9993C38 40.8653 34.866 43.9993 31 43.9993C27.134 43.9993 24 40.8653 24 36.9993Z" fill="#C5C9FD" class="icon-light-secondary"/>
<path fill-rule="evenodd" clip-rule="evenodd" d="M30 30.9993C30 30.447 30.4477 29.9993 31 29.9993C34.866 29.9993 38 33.1333 38 36.9993C38 37.5516 37.5523 37.9993 37 37.9993C36.4477 37.9993 36 37.5516 36 36.9993C36 34.2378 33.7614 31.9993 31 31.9993C30.4477 31.9993 30 31.5516 30 30.9993Z" fill="#4956E3" class="icon-dark-secondary"/>
<path d="M36 36.9993C36 39.7607 33.7614 41.9993 31 41.9993C28.2386 41.9993 26 39.7607 26 36.9993C26 34.2378 28.2386 31.9993 31 31.9993C33.7614 31.9993 36 34.2378 36 36.9993Z" fill="white"/>
```